pkgs/applications/virtualization/lima/default.nix

```diff
@@ -39,10 +39,19 @@ buildGoModule rec {
 
   buildPhase = ''
     runHook preBuild
-    make "VERSION=v${version}" binaries
+    make "VERSION=v${version}" "CC=${stdenv.cc.targetPrefix}cc" binaries
     runHook postBuild
   '';
 
+  preCheck =
+    if stdenv.hostPlatform.isLinux then
+      ''
+        # Workaround for: could not create "/homeless-shelter/.lima/_config" directory: mkdir /homeless-shelter: permission denied
+        export LIMA_HOME="$(mktemp --directory)"
+      ''
+    else
+      null;
```

Comment:
Let’s perhaps just do this unconditionally, if it doesn’t harm anything on macOS?